app/views/admin/accounts/index.html.haml

```diff
@@ -31,6 +31,12 @@
         - if account.user.homesite
           %br
           = account.user.homesite
+        - if account.user.jabber_id
```

Comment:
For your information, with Rails, you can add `.present?` to test if string is neither Nil nor empty string.

If user remove its jabber_id, it will be an empty string and the <br> tag will be shown.

To avoid to show the br tag, you can use:

```diff
-      - if account.user.jabber_id
+      - if account.user.jabber_id.present?
```

Reply:
done, thanks